release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -2959,9 +2966,9 @@ To revert all nodes in the cluster to the cgroups v2 configuration, you must edi
 +
 You can control IP forwarding for all traffic on OVN-Kubernetes managed interfaces by using the `gatewayConfig.ipForwarding` specification in the `Network` resource. Specify `Restricted` to forward all traffic related to OVN-Kubernetes only. Specify `Global` to allow forwarding of all IP traffic. For new installations, the default is `Restricted`. For upgrades to {product-version}, the default is `Global`. (link:https://issues.redhat.com/browse/OCPBUGS-3176[*OCPBUGS-3176*]) (link:https://issues.redhat.com/browse/OCPBUGS-16051[*OCPBUGS-16051*])
 
-* For clusters that have root volume availability zones and are running on {rh-openstack} that you upgrade to {product-version} and have root volume availability zones, you must converge control plane machines onto one server group before you can enable control plane machine sets. To make the required change, follow the instructions in the link:https://access.redhat.com/solutions/7013893[Knowledgebase article]. (link:https://issues.redhat.com/browse/OCPBUGS-13300[*OCPBUGS-13300*])
+* For clusters that have root volume availability zones and are running on {rh-openstack} that you upgrade to {product-version} and have root volume availability zones, you must converge control plane machines onto one server group before you can enable control plane machine sets. To make the required change, follow the instructions in the link:https://access.redhat.com/solutions/7013893[knowledge base article]. (link:https://issues.redhat.com/browse/OCPBUGS-13300[*OCPBUGS-13300*])
```

Comment:
We don't write ours as two words

See https://redhat-documentation.github.io/supplementary-style-guide/#knowledgebase and https://redhat-documentation.github.io/supplementary-style-guide/#rh-kb-links